src/gui/gui2_advancedscrolltext.cpp

```diff
@@ -44,40 +48,51 @@ GuiAdvancedScrollText* GuiAdvancedScrollText::clearEntries()
 
 void GuiAdvancedScrollText::onDraw(sp::RenderTarget& renderer)
 {
-    //For all the entries, fix the maximum prefix width, so we know how much space we have for the text.
-    float max_prefix_width = 0.0f;
-    for(Entry& e : entries)
-    {
-        auto prepared = sp::RenderTarget::getDefaultFont()->prepare(e.prefix, 32, text_size, {0, 0}, sp::Alignment::TopLeft, 0);
-        max_prefix_width = std::max(max_prefix_width, prepared.getUsedAreaSize().x);
-    }
+    const bool is_resized = rect_width != rect.size.x;
+    if (is_resized) { rect_width = rect.size.x; }
 
     //Draw the visible entries
-    float draw_offset = -scrollbar->getValue();
+    float draw_offset = -scrollbar->getValue() + text_size + 12.0f;
```

Comment:
This is a bad kludge to realign text now that the positioning lives in the `Entry` rather than recalculated each onDraw.